src/Orckestra.Composer.Website/App_Data/Razor/Composer/Product/Summary.cshtml

```diff
@@ -67,64 +68,114 @@
-{
-    if (IsWithoutVariants())
-    {
-        <h1>@Model.DisplayName</h1>
-    }
-    else
-    {
-        var displayNames = Model.Variants.GroupBy(v => v.DisplayName).ToList();
+        </body>
+        </html>
 
-        foreach (var group in displayNames)
+        @helper DisplayName()
         {
-            var ids = group.Select(v => v.Id).ToArray();
-            <span data-variant="@(String.Join(",", ids))" class="@SelectedVariantClass(ids) h1">@group.Key</span>
+            if (IsWithoutVariants())
+            {
+                <h1>@Model.DisplayName</h1>
+            }
+            else
+            {
+                var displayNames = Model.Variants.GroupBy(v => v.DisplayName).ToList();
+
+                foreach (var group in displayNames)
+                {
+                    var ids = group.Select(v => v.Id).ToArray();
+                    <span data-variant="@(String.Join(",", ids))" class="@SelectedVariantClass(ids) h1">@group.Key</span>
+                }
+                <h1 data-variant="unavailable" class="d-none">@Model.DisplayName</h1>
+            }
         }
-        <h1 data-variant="unavailable" class="d-none">@Model.DisplayName</h1>
-    }
-}
 
-@helper Description()
-{
-<div class="mb-4" property="description">
-    @HttpUtility.HtmlDecode(Model.Description)
-</div>
-}
+        @helper Description()
+        {
+        <div class="mb-4" property="description">
+            @HttpUtility.HtmlDecode(Model.Description)
+        </div>
+        }
 
-@helper Sku()
-{
-    <span class="d-none" property="sku">@Model.Sku</span>
-}
+        @helper Sku()
+        {
+            <span class="d-none" property="sku">@Model.Sku</span>
+        }
 
-@helper BrandName()
-{
-    <h2 class="h5  mb-1" property="brand" data-qa="product-brand">@HttpUtility.HtmlDecode(Model.Brand)</h2>
-}
+        @helper BrandName()
+        {
+            <h2 class="h5  mb-1" property="brand" data-qa="product-brand">@HttpUtility.HtmlDecode(Model.Brand)</h2>
+        }
 
-@helper StoreInventoryLink()
-{
-    <div class="mb-4">
-        <div class="row">
-            <div class="col-12 col-lg-7">
-                <a class="btn btn-outline-default btn-block mb-4 relative" href="#storeinventory">
-                    <i class="fa  fa-store"></i>&#160;
-                    @Html.Localize("Store", "H_StoreAvailability")
-                </a>
+        @helper StoreInventoryLink()
+        {
+            <div class="mb-4">
+                <div class="row">
+                    <div class="col-12 col-lg-7">
+                        <a class="btn btn-outline-default btn-block mb-4 relative" href="#storeinventory">
+                            <i class="fa  fa-store"></i>&#160;
+                            @Html.Localize("Store", "H_StoreAvailability")
+                        </a>
+                    </div>
+                </div>
             </div>
-        </div>
-    </div>
-}
\ No newline at end of file
+        }
+
+        @helper KvaList()
```

Comment:
is it used? looks like you have commented    @* @KvaList() *@

Reply:
not used